dev/ubuntu22.04.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:22.04
+WORKDIR /home/
+
+RUN ["apt-get", "-y", "upgrade"]
+RUN ["apt-get", "-y", "update"]
```

Comment:
you might want to stuff build-essential in here (although I suppose the image probably already has it)

Reply:
It does not come by default, I popped it into the setup script because it's on the list of 'things you need to download to get ghcup working'. I can also put it here, with a wary eye toward the possibility that we could iterate for a long time about what a reasonable base Dockerfile looks like and I don't want to fall down that rabbit hole right now 🐰

Reply:
Probably better off in the setup script. Anyone who doesn't have it is going to want it.